ChainRouter.php

```diff
@@ -306,4 +306,25 @@ public function getRouteCollection()
 
         return $this->routeCollection;
     }
+    
+    /**
+     * Identify if any routers have been added
+     * 
+     * @return bool
+     */
+    public function hasRouters()
+    {
+        return !empty($this->routers);
+    }
+    
+    /**
+     * Get the number of added routers
+     * 
+     * @return integer
+     */
+    public function getNumRouters()
```

Comment:
Well - hasRouters() is useful, but it still doesn't tell you *how many* there are. So getNumRouters() adds more value than hasRouters() in my opinion since you can do ``` if(getNumRouters() > 0)```